src/main/java/com/toptrumps/core/Dealer.java

```diff
@@ -1,4 +1,30 @@
 package com.toptrumps.core;
 
```

Comment:
There is no import for the ArrayList? For me, it won't compile.

Reply:
Oops my bad. I got this when I'd compiled for my test and forgot to push the update! Should be fine now